docs/alma/alma.rst

```diff
@@ -294,6 +294,55 @@ their types.
       velocity_resolution  double          m/s        Estimated velocity resolution from all the spectral windows, from frequency resolution.
 
 
+Query Results
+=============
+
+Results of queries are returned in tabular format. For convenience,
+the ''to_enhanced_table'' function can be used to have the initial result
+in a more useful format, i.e turn values into quantities, footprint into
+shape, etc. (Note: this require the ''shapes'' Python package to be installed.
+
+
+.. doctest-remote-data::
+
+    >>> from astroquery.alma import Alma, to_enhanced_table
+    >>> alma = Alma()
+    >>> alma.archive_url = 'https://almascience.eso.org'  # optional to make doctest work
+    >>> res = alma.query_tap("select top 1 * from ivoa.ObsCore where obs_publisher_did='ADS/JAO.ALMA#2021.1.00547.S'")
+    >>> enhanced_res = to_enhanced_table(res)
```

Comment:
From this example, it looks like `to_enhanced_table` should be renamed.  Maybe it should be `enhance_table(...)`?  `to_enhanced_table` sounds to me like an object method, not a standalone function.

Reply:
You are perfectly right. It used to be a method. Changed now to `get_enhanced_table`. Was also thinking of `extract_enhnaced_table` as an alternative